packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -31,26 +30,18 @@ const selectors = {
 	saveDraftButton: '.editor-post-save-draft',
 	saveDraftDisabledButton: 'button.editor-post-saved-state.is-saved[aria-disabled="true"]',
 	previewButton: ':is(button:text("Preview"), a:text("Preview"))',
-	publishButton: ( parentSelector: string ) =>
-		`${ parentSelector } button:text("Publish")[aria-disabled=false]`,
-	updateButton: 'button:text("Update")',
+	publishButton: `.editor-post-publish-button__button`,
+	// publishButton: ( parentSelector: string ) =>
+	// `${ parentSelector } button:text("Publish")[aria-disabled=false]`,
 	switchToDraftButton: 'button.editor-post-switch-to-draft',
-	scheduleButton: ( parentSelector: string ) =>
-		`${ parentSelector } button:has-text("Schedule")[aria-disabled=false]`,
+	// scheduleButton: ( parentSelector: string ) =>
+	// 	`${ parentSelector } button:has-text("Schedule")[aria-disabled=false]`,
```

Comment:
Heads up: probably meant to delete these commented lines?

Reply:
Yeah, probably. Oops.